src/components/ApplyFiltersButton.tsx

```diff
@@ -1,29 +1,44 @@
 import { useAnswersActions } from '@yext/answers-headless-react';
 import { useCallback } from 'react';
-import { executeSearch } from '../../utils/search-operations';
+import { executeSearch } from '../utils/search-operations';
 
 /**
- * Props for {@link Filters.ApplyFiltersButton}
+ * Props for {@link ApplyFiltersButton}
  *
  * @public
  */
 export interface ApplyFiltersButtonProps {
-  /** The css classes for the button */
-  className?: string,
   /** The label for the button, defaults to 'Apply Filters' */
-  label?: string
+  label?: string,
+  /** CSS classes for customizing the component styling. */
+  customCssClasses?: ApplyFiltersButtonCssClasses
 }
 
+
+/**
+ * The CSS class interface for {@link ApplyFiltersButtonProps}.
+ *
+ * @public
+ */
+export interface ApplyFiltersButtonCssClasses {
+  button?: string
```

Comment:
did we change this to an interface instead of a single className because we anticipate adding more CSS classes to the component?

Reply:
It's to be more consistent with the rest of the components